code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1684,6 +1684,13 @@ var/global/list/obj/item/device/pda/PDAs = list()
 					if (mode == 1)
 						note = replacetext(n, "\n", "<BR>")
```

Comment:
Though I get what you were saying and adjusted it so it's not replacetexting note and just does it off n